CONTRIBUTING.md

```diff
@@ -0,0 +1,106 @@
+# Contributing to F#
```

Comment:
I propose we add a checkbox and link to this page to the .github/pull_request_template:

- [ ] I have read and understood the [CONTRIBUTING](https://github.com/dotnet/fsharp/blob/main/CONTRIBUTING.md) guide

Reply:
Nah!!!!

Reply:
Few developers are one time only contributors.  There is no need to TAX everybody, even though the TAX is an extra single click.

Reply:
We do have 3 checkboxes already (tests, benchmarks, release notes).
This page is more important to be regularily reminded than at least 2 out of the 3 existing ones.

Reply:
Up for adding a tick there.